Tools/ardupilotwaf/boards.py

```diff
@@ -231,6 +231,13 @@ def configure_env(self, cfg, env):
                       args = ['--libs', '--cflags'])
         env.LIB += cfg.env['LIB_LTTNG-UST']
 
+        cfg.check_cfg(package='libiio', mandatory=False, global_define=True,
+                      args = ['--libs', '--cflags'])
```

Comment:
This is still a wip. I may decide to submit these waf-related patches as a separate PR later.